components/gitops/staging/base/authentication/gitops-rolebindings.yaml

```diff
@@ -0,0 +1,58 @@
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: gitops-namespace-all-access
+  namespace: gitops
+subjects:
+  - kind: User
```

Comment:
We are syncing groups from redhat-appstudio teams -> so maybe this could be used https://github.com/orgs/redhat-appstudio/teams/gitops-service-team ?
Example: https://github.com/redhat-appstudio/infra-deployments/blob/main/components/authentication/base/internal-services.yaml#L7-L9

Reply:
:+1: from me